admin_app/src/app/page.tsx

```diff
@@ -195,10 +209,9 @@ export default function Home() {
                     />
                   </div>
                   <div className="relative p-2 flex flex-auto">
-                    <textarea
+                    <TextareaAutosize
                       id="content_text"
-                      rows={16}
```

Comment:
Can we keep rows to say 4 or 6?
Also, can we set max rows before it switches to a scroll bar?